app/Lib/Rscripts/analyse.R

```diff
@@ -2,11 +2,45 @@
 library(psy)
 
 Analyse <- function(key, input.answers, number.answeroptions) {
+  # Calculates standard psychometric properties of an exam, notably the 
+  # percentage correct and the item rest correlation (IRC) per item and
+  # per answer option and the cronbach's alpha for the whole exam.
+  #
+  # Arguments:
```

Comment:
I think I forgot to sync. I synced it just now. I am not sure, but I think I do not need to do a new pull request as the changes seem to have been added to the current pull request as the appear below.